contributors/design-proposals/core-metrics-pipeline.md

```diff
@@ -0,0 +1,315 @@
+# Core Metrics Pipeline in kubelet
+
+**Author**: David Ashpole (@dashpole)
+
+**Last Updated**: 12/21/2016
+
+**Status**: Draft Proposal (WIP)
+
+This document proposes a design for an internal Core Metrics Pipeline.
+
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+
+- [Core Metrics Pipeline in kubelet](#core-metrics-pipeline-in-kubelet)
+  - [Introduction](#introduction)
+    - [Background](#background)
+    - [Motivations](#motivations)
+    - [Proposal](#proposal)
+    - [Non Goals](#non-goals)
+  - [Design](#design)
+    - [Proposed Core Metrics API:](#proposed-core-metrics-api)
+    - [Core Machine Info:](#core-machine-info)
+  - [Implementation Plan](#implementation-plan)
+  - [Rollout Plan](#rollout-plan)
+  - [Implementation Status](#implementation-status)
+
+<!-- END MUNGE: GENERATED_TOC -->
+
+## Introduction
+
+### Background
+The [Monitoring Architecture](https://github.com/kubernetes/kubernetes/blob/master/docs/design/monitoring_architecture.md) proposal contains a blueprint for a set of metrics referred to as "Core Metrics".  The purpose of this proposal is to specify what those metrics are, and how they will be collected on the node.
```

Comment:
This was meant to be a proposal for the core metrics portion.  Are you saying that we should expose system metrics rather than just core metrics through any external URL endpoint?